lib/language_syntax/golang_syntax.dart

```diff
@@ -0,0 +1,57 @@
+// Обработка правильности отступов, точек с запятой для цикла for
+// с учетом комментариев, с учетом строк.
+Map<int, String> findGolangErrors(String text) {
+  List<String> lines = text.split("\n");
+  Map<int, String> errors = {};
+  int indentLevel = 0;
+
+  for (int i = 0; i < lines.length; i++) {
+    String currentLine = lines[i];
+
+    if (currentLine.trim().isEmpty ||
+        currentLine.startsWith(RegExp("\\s*//"))) {
+      continue;
+    }
+
+    if (currentLine.startsWith(RegExp("\\s*/\\*"))) {
+      while ((!currentLine.contains(RegExp("\\*/\\s*"))) &&
+          (i < lines.length - 1)) {
+        i++;
+        currentLine = lines[i];
+      }
+    } else if (currentLine.startsWith(RegExp(".*`"))) {
```

Comment:
move else/else if   to next line everywhere

Reply:
I don't do that, but the code reformer built into android studio.

Reply:
I don't mind having it this way. Just make sure that the style is the same throughout the project